### PR TITLE
Migrate to Github actions

### PR DESCRIPTION
**Pull request by cmodijk**
What is inside this pull request
- I moved all the packages to the `packages/` folder to allow `symplify/monorepo-builder` to test all the packages with there own unit tests and requirements
- I made sure al the requirements between de packages are in sync
- I removed the `.travis.yml` files but we still need to remove the integrations hooks   
- For testing i added the `minimum-stability` and `prefer-stable` settings
- I changed the tests from Symfony  `4.4.*` and `5.0.*` to `4.4.*` and `5.2.*`
- Changed `Doctrine\Common\Persistence\ManagerRegistry` to `Doctrine\Persistence\ManagerRegistry`
- Fixed the unit tests for `rabbitmq-bundle-bridge`
- Added PHP `8.0` as a matrix 
- Fixed the first PHP `8.0` tests

TODO
- [x] Make sure we `--ignore-platform-req php` is only used in the `8.0` version (for now)
- [x] Find a way to only start the RabbitMQ service when needed
- [x] Make sure we add back a case where we test the `prefer-lowest` test case
- [x] Determine what to do with the failing `rabbitmq-bundle-bridge` on PHP `8.0`
   *https://github.com/php-amqplib/php-amqplib/issues/846*
- [x] https://github.com/php-amqplib/RabbitMqBundle/pull/605
- [x] Fix new test errors
- [x] Remove Travis webhooks
- [x] Bring back the `REMOVE_PROXY_BRIDGE` test case or check why it was needed
- [ ] Use monorepobuilder merge (blocked by https://github.com/symplify/symplify/pull/2605)
- [ ] Run tests also on the monorepo (this tests everything at once), instead of only per package
- [ ] Add split action

### CODE REVIEW

**Comment by ruudk** [comment]
The `authors` section is removed and that's fixed by https://github.com/symplify/symplify/pull/2605. Waiting for that to be merged and tagged before I can continue with LongRunning.

**Comment by cmodijk** [comment]
Yeah i removed it manually i first want the build to work before using the merge

**Comment by cmodijk** [comment]
I will get back to this tomorrow

File: .github/workflows/tests.yaml
Comment (ruudk) [inline .github/workflows/tests.yaml:41]: Given the size of the matrix, maybe we should just always use `lowest`?

File: .github/workflows/tests.yaml
Comment (ruudk) [inline .github/workflows/tests.yaml:35]: We can do this in a future PR, but I think it's nice to build this dynamically, using:
https://github.com/LongRunning/LongRunning/blob/main/.github/workflows/tests.yaml#L21-L22
https://github.com/LongRunning/LongRunning/blob/main/.github/workflows/tests.yaml#L25
https://github.com/LongRunning/LongRunning/blob/main/.github/workflows/tests.yaml#L33

**Review by ruudk — COMMENTED** [review]
Great work @cmodijk 🙌

File: .github/workflows/tests.yaml
Comment (cmodijk) [reply .github/workflows/tests.yaml:35]: I installed `vendor/bin/easy-ci` but it gave me a error  i have to look into this

File: .github/workflows/tests.yaml
Comment (cmodijk) [reply .github/workflows/tests.yaml:41]: I think we should also always test agains the latest versions.

File: .github/workflows/tests.yaml
Comment (cmodijk) [reply .github/workflows/tests.yaml:35]:
```bash
./vendor/bin/easy-ci php-versions-json
```
```output                                                                                                                  
 [ERROR] Class 'Symplify\PHPStanRules\Exception\ShouldNotHappenException' not found    
```
```json
{
    "name": "simple-bus/simple-bus",
    "description": "Simple CommandBus and EventBus implementation",
    "license": "MIT",
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/mihaileu/RabbitMqBundle.git"
        }
    ],
    "autoload-dev": {
        "psr-4": {
            "SimpleBus\\CI\\": "src"
        }
    },
    "require-dev": {
        "php": "^7.3 || ^8.0",
        "ext-json": "*",
        "symplify/easy-ci": "9.0.0-rc1",
        "symplify/monorepo-builder": "9.0.0-rc1"
    },
    "config": {
        "sort-packages": true
    },
    "minimum-stability": "dev",
    "prefer-stable": true,
    "require": {
        "symfony/console": "^4.4 || ^5.0"
    }
}
```

File: .github/workflows/tests.yaml
Comment (cmodijk) [reply .github/workflows/tests.yaml:41]: I updated the exclude to only run once

**Comment by cmodijk** [comment]
@ruudk I looked into the `REMOVE_PROXY_BRIDGE` case and it tries to test this one case / check inside the code. I'm not sure if we should just just remove this test, what do you think? 
*https://github.com/SimpleBus/SimpleBus/blob/master/Bundle/SymfonyBridge/tests/Functional/SmokeTest.php#L138-L148*

**Review by ruudk — DISMISSED** [review]

**Comment by cmodijk** [comment]
@ruudk I rebased this on top op the latest `master` so that is why the review got removed. I went along and still merged this so it is in `master` right now.

**Comment by holtkamp** [comment]
Would it be possible to tag a release which supports PHP 8?

For example, dev-master https://github.com/SimpleBus/Asynchronous/blob/d7d22d4d7a62241d72b90f83e075af70ca98efe4/composer.json#L29 allows it, but there is no way to install it...

**Comment by cmodijk** [comment]
I still want to finish https://github.com/SimpleBus/SimpleBus/pull/106 but i currently don't have a lot of time for this.

@ruudk Have you already tested the latest version on PHP 8? If so then i don't see a problem in tagging the first version.

**Comment by ruudk** [comment]
Unfortunately we're not on PHP 8 yet because we're blocked by NewRelic :(.

I don't think we have to wait for #106 as that's just nice to have. 

@holtkamp Are you able to test current `dev-master`?

**Comment by holtkamp** [comment]
>Unfortunately we're not on PHP 8 yet because we're blocked by NewRelic :(.

Same here...

>@holtkamp Are you able to test current dev-master?

Yes, been using it in production on PHP `7.4.13` for some days without encountering any issues using the follwing `composer.json` configuration:

```
    "simple-bus/bernard-bundle-bridge": "dev-allow-php-8",
    "simple-bus/asynchronous": "dev-master as 3.1.1",
    "simple-bus/asynchronous-bundle": "dev-master as 3.2.1",
    "simple-bus/message-bus": "dev-master as 3.1.1",
    "simple-bus/serialization": "dev-master as 3.1.1",
```
Locally it is running on PHP `8.0.1` without encountering any issues...

**Comment by ruudk** [comment]
Sounds good :) @cmodijk Could you take a new release?

**Pull request merged**
